src/main/java/de/tum/in/www1/artemis/service/connectors/localvc/LocalVCInfoContributor.java

```diff
@@ -43,7 +43,7 @@ public void contribute(Info.Builder builder) {
         // TODO: only activate this when access tokens are available and make sure this does not lead to issues
         // TODO: If activated, reflect this in LocalVCInfoContributorTest
         // with the account.service.ts and its check if the access token is required
```

Comment:
Are they now resolved? Should they be updated?

Reply:
We will keep it disabled for now most likely